ingestion_tools/scripts/common/finders.py

```diff
@@ -185,45 +226,55 @@ def _get_results(
         for name, path in found.items():
             name_str = str(name)  # Wrapper to prevent float voxel spacings from erroring
             if any(exclude_regex.match(name_str) for exclude_regex in self.exclude_regexes):
-                print(f"Excluding {cls.type_key} {name}...")
+                print(f"Excluding {self.importer_cls.type_key} {name}...")
                 continue
             filtered_results[name] = path
-        return self._get_instantiated_results(cls, config, metadata, filtered_results, parent_objects)
+        return self._get_instantiated_results(config, metadata, filtered_results, loader.allow_imports, parent_objects)
 
     def _get_instantiated_results(
         self,
-        cls,
         config: DepositionImportConfig,
         metadata: dict[str, Any],
         filtered_results: dict[str, str],
+        allow_imports: bool,
         parents: dict[str, Any] | None,
     ) -> list[BaseImporter]:
         results = []
         for name, path in filtered_results.items():
-            item = self._instantiate(cls, config, metadata, name, path, parents)
+            name, path, metadata = self.handle_for_metadata_file(config, name, path, metadata)
+            item = self._instantiate(config, metadata, name, path, allow_imports, parents)
```

Comment:
Again with special casing for the `*_metadata.json` files. Can we at least get some comments (and tests!) that cover this special case?